createdisk.sh

```diff
@@ -131,7 +131,7 @@ create_tarball "$libvirtDestDir"
 # the content of $libvirtDestDir
 if [ -n "${SNC_GENERATE_MACOS_BUNDLE}" ]; then
     hyperkitDestDir="crc_hyperkit_${destDirSuffix}"
-    generate_hyperkit_bundle "$libvirtDestDir" "$hyperkitDestDir" "$1" "$kernel_release" "$kernel_cmd_line"
+    generate_hyperkit_bundle "$libvirtDestDir" "$hyperkitDestDir" "$1" "$kernel_release" "$kernel_cmd_line" "$podman_version"
```

Comment:
Would be better to let `download_podman` handle this rather than carrying around the podman version. This is 4.9-only isn't this?

Reply:
@cfergeau you mean `download_podman` should download => extract => put the podman executable in top dir? 

Yes it is only 4.9 side.

Reply:
> @cfergeau you mean download_podman should download => extract => put the podman executable in top dir?

Yup, something like this:
```
diff --git a/createdisk-library.sh b/createdisk-library.sh
index 25f4bda..a515614 100755
--- a/createdisk-library.sh
+++ b/createdisk-library.sh
@@ -294,11 +294,13 @@ function download_podman() {
       mkdir -p podman-remote/mac
       curl -L https://github.com/containers/podman/releases/download/v${version}/podman-remote-release-darwin.zip -o podman-remote/mac/podman.zip
       ${UNZIP} -o -d podman-remote/mac/ podman-remote/mac/podman.zip
+      mv podman-remote/mac/podman-${podman-version}/podman  podman-remote/mac
     fi
 
     if [ -n "${SNC_GENERATE_WINDOWS_BUNDLE}" ]; then
       mkdir -p podman-remote/windows
       curl -L https://github.com/containers/podman/releases/download/v${version}/podman-remote-release-windows.zip -o podman-remote/windows/podman.zip
       ${UNZIP} -o -d podman-remote/windows/ podman-remote/windows/podman.zip
+      mv podman-remote/windows/podman-${podman-version}/podman  podman-remote/windows
     fi
 }
```